articles/hooks/extensibility-points/send-phone-message.md

```diff
@@ -0,0 +1,108 @@
+---
+description: Learn how hooks can be used with the Send Phone Message extensibility point, which is available for MFA.
+beta: true
+toc: true
+topics:
+    - hooks
+    - extensibility-points
+    - send-phone-message
+    - custom-sms-gateway
+    - sms 
+contentType:
+  - how-to
+useCase: extensibility-hooks
+v2: true
+---
+# Send Phone Message
```

Comment:
At the end we should add a 'Keep Reading' section that points to the Amazon example

Reply:
done